spec/requests/api/v1/rooms_spec.rb

```diff
@@ -177,4 +177,18 @@
       end
     end
   end
+
+  describe "GET /api/v1/rooms/recommend" do
+    let(:recommend) { create(:recommend_room) }
```

Comment:
```suggestion
    let!(:recommend) { create(:recommend_room) }
```
こうすれば下のbefore要らなさそう？